apps/desktop/src/components/v3/NewCommit.svelte

```diff
@@ -69,6 +69,7 @@
 			stackId,
 			parentId,
 			message: message,
+			stackSegmentShortName: "top",
```

Comment:
Here is where the actual stack-segment name needs to be added.

<img width="305" alt="Screenshot 2025-02-27 at 15 33 55" src="https://github.com/user-attachments/assets/6714f09e-ad86-4d9e-9104-3222d0b98f9d" />

Where ever the "Your commit goes here" marker is, that should inform the name to be passed. In the case above, it would be "top".

That way, the commit will always go into the right segment/bucket.